crates/model/src/signature.rs

```diff
@@ -35,9 +36,19 @@ impl From<QuoteSigningScheme> for SigningScheme {
 #[derive(Eq, PartialEq, Clone, Deserialize, Serialize, Hash)]
 #[serde(into = "JsonSignature", try_from = "JsonSignature")]
 pub enum Signature {
+    /// Transaction data hashed and signed according to EIP-712.
+    /// https://eips.ethereum.org/EIPS/eip-712
     Eip712(EcdsaSignature),
+    /// A standard Ethereum transaction signature.
     EthSign(EcdsaSignature),
+    /// Transaction data signed according to EIP-1271, which facilitates a way for contracts to
+    /// verify signatures using an arbitrary method. This allows smart contracts to sign and
+    /// place orders.
+    /// https://eips.ethereum.org/EIPS/eip-1271
     Eip1271(Vec<u8>),
+    /// Data signed using EIP-191. Similar to the [`Self::Eip712`] variant, except the data is not
+    /// structured.
+    /// https://eips.ethereum.org/EIPS/eip-191
     PreSign,
```

Comment:
`PreSign` doesn't use EIP-191 but it's a custom implementation that relies on onchain transactions, so there is no real reference to a specification here. Note however that the order hash used here is derived from parts of the EIP-712 specification as for all other signing schemes.
EIP-191 is technically used for `EthSign` however (see `personal_sign` messages).